src/main/java/com/nitnelave/CreeperHeal/block/CreeperMultiblock.java

```diff
@@ -38,18 +38,18 @@ public void update()
     @Override
     protected boolean checkForDrop()
     {
-        if (checkForDropHelper(getBlock()))
```

Comment:
@nitnelave I'd appreciate your input - I don't see a clean way to accomplish your [original request](https://github.com/nitnelave/CreeperHeal/pull/61#discussion_r221838690).

>Remove this function, and turn the` checkForDrop` function in CreeperBlock static, it's the same.

We need to be able to call `CreeperBlock#drop` in a non-static context so we have access to the original BlockState in the event that the state should be the block broken, not the current block.

Reply:
Sorry, I lost a bit of context over the weeks :D 

From what I can see, you don't use any non-static variable in the checkForDrop function: you can then make it independent from the instance, and make it static. You can still call b.drop(), that's not a problem; you're just not using "this".

Reply:
The issue is that the BlockState for which the drops are being generated may be the state stored in the CreeperBlock or the in-world block depending on the config options in use. We need to be able to access the CreeperBlock instance to be able to drop its stored state.

Reply:
oh, right, I missed this line, sorry. Go ahead, then.